File: Tareas/Unidad2/a2253332351_practica01/02_CantArena.java
import java.util.Scanner;

public class CantArena {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double metros, largo, ancho, arena;

            System.out.println("Ingrese el largo de la pared en metros: ");
            largo = entrada.nextDouble();
            System.out.println("Ingrese el ancho de la pared en metros: ");
            ancho = entrada.nextDouble();

            metros = 0.5;
            arena = largo * ancho * metros;

            System.out.println("La cantidad de arena necesaria es de: " + arena);
     }
}

File: Tareas/Unidad2/a2253332351_practica01/02_CantArena.cpp
#include <iostream>
using namespace std;

int main() {
    double metros = 0.5;
    double largo;
    double ancho;
    double arena;

    cout << "Ingrese el largo de la pared en metros: ";
    cin >> largo;
    cout << "Ingrese el ancho de la pared en metros: ";
    cin >> ancho;

    arena = largo * ancho * metros;

    cout << "La cantidad de arena necesaria es de: " << arena << endl;

    return 0;
}
